docker/app/run-celery.sh

```diff
@@ -5,3 +5,18 @@
   worker \
   --concurrency="${QUEUE_WORKERS}" \
   --loglevel="${LOKOLE_LOG_LEVEL}"
+  --Q="written_store"
+
+"${PY_ENV}/bin/celery" \
```

Comment:
In Docker it's usually advisable to have a single process per container (see more information here https://docs.docker.com/config/containers/multi-service_container/). As such, it would be great if the queue name could be configured via an environment variable so that we can start one container per queue and keep a single process per container.

Related question: is there a way to make the Celery worker listen to multiple queues using this syntax? It would be great if we had support for that so that we can either run one container for all queues (easier for debugging) or one container per queue (better for production). In any case, there will still be only one process in the container since we'd only be starting a single executable in the run script.

Reply:
Ok, noted. The -Q option can take multiple queues (-Q q1, q2....) I believe just specifying all three queues will make 1 worker listen to all of them.